src/main/java/com/woowacamp/soolsool/core/liquor/controller/LiquorController.java

```diff
@@ -93,21 +90,45 @@ public ResponseEntity<ApiResponse<PageLiquorResponse>> liquorList(
         @RequestParam("region") @Nullable final LiquorRegionType region,
         @RequestParam("status") @Nullable final LiquorStatusType status,
         @RequestParam("brand") @Nullable final String brand,
-        @RequestParam @Nullable final Long cursorId,
+        @RequestParam @Nullable final Long liquorId,
         @PageableDefault final Pageable pageable
     ) {
-        final PageRequest sortPageable = PageRequest.of(
-            pageable.getPageNumber(),
-            pageable.getPageSize(),
-            Sort.by("createdAt").descending()
-        );
+        final PageRequest sortPageable = getSortedPageable(pageable);
 
         final PageLiquorResponse response = liquorService
-            .liquorList(brew, region, status, brand, sortPageable, cursorId);
+            .liquorListByLatest(brew, region, status, brand, sortPageable, liquorId);
 
         return ResponseEntity.ok(ApiResponse.of(LIQUOR_LIST_FOUND, response));
     }
 
+    @NoAuth
+    @RequestLogging
+    @GetMapping("/click")
+    public ResponseEntity<ApiResponse<PageLiquorClickResponse>> liquorListByClick(
+        @RequestParam("brew") @Nullable final LiquorBrewType brew,
+        @RequestParam("region") @Nullable final LiquorRegionType region,
+        @RequestParam("status") @Nullable final LiquorStatusType status,
+        @RequestParam("brand") @Nullable final String brand,
+        @RequestParam @Nullable final Long liquorId,
+        @RequestParam @Nullable final Long clickCount,
+        @PageableDefault final Pageable pageable
+    ) {
+        final PageRequest sortPageable = getSortedPageable(pageable);
```

Comment:
```suggestion
        final PageRequest sortPageable = PageRequest.createdAtDesc(pageable);
```

정적 팩터리 메서드에 이름을 붙여서 private 메서드와 매직 넘버를 다른 곳에서 관리할 수 있을 것 같은데 어떻게 생각하시나요?

Reply:
페이지 리퀘스트는 제가 정의한게 아닌데, 상속해서 새로운 객체 생성하여 메서드를 만들라는 말씀이실까요???

Reply:
```java
    private  PageRequest getSortedPageable(Pageable pageable) {
        return PageRequest.of(
            pageable.getPageNumber(),
            pageable.getPageSize(),
            Sort.by("createdAt").descending()
        );
    }
```

아뇨 바로 아래 부분에 있는 코드를 DTO 클래스로 이동하면 어떨까? 라는 생각이었습니다.

정적 팩토리 메서드 패턴의 장점인 생성자에 유의미한 이름을 붙일 수 있다를 of, from으로만 활용하는게 아쉬워서요!

Reply:
어떤 dto말씀하시는 걸까요?? PageRequest는 Pageable 에서 제공하는 클래스입니다!